Web/starter_project_vanguard/components/story/Partners.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+const Partners = () => {
+  return (
+    <div className="max-w-screen-lg mt-20">
+      <h1 className="font-DMSans text-5xl font-semibold mb-16 text-center">
+        Current Interview Partners
+      </h1>
+      <div className="flex flex-col sm:flex-row  gap-1">
+        <img
+          src="story_img/Google.png"
+          alt="Google logo"
+          className="w-full max-w-screen-lg sm:max-w-none"
+        />
+        <img
+          src="story_img/Palantir.png"
+          alt="Palantir logo"
+          className="w-full max-w-screen-lg sm:max-w-none"
+        />
+        <img
+          src="story_img/InstDeep.png"
+          alt="InstaDeep logo"
+          className="w-full max-w-screen-lg sm:max-w-none"
+        />
+        <img
+          src="story_img/meta.png"
+          alt="Meta logo"
+          className="w-full max-w-screen-lg sm:max-w-none"
+        />
+      </div>
+      <div className="flex gap-1 flex-col sm:flex-row justify-between mx-0 md:mx-40">
```

Comment:
not sure why we have the databricks and linkedin logo inside a div separate from the others
if their sizing is weird, it is probably because the images are cropped werid and can be fixed by cropping them appropriately instead of modifying code